### DISCUSSION TITLE
Separate Dataset

### DISCUSSION THREAD

**Discussion by ageljonathan** [teamalgoritma/community#455] [Q&A] [answered]
Selamat siang
Saya ingin bertanya

Bagaimana cara memisahkan dataset seperti ini menjadi multiple kolom
![Capture](https://user-images.githubusercontent.com/112919414/202615062-8634ca4b-0041-4652-b1f9-32c1965df289.JPG)

Terima kasih

**Comment by kevbow** [comment, marked as answer]
Halo @ageljonathan ,
Pada fungsi `read.csv()` terdapat parameter `sep` yang dapat diisi dengan pemisah selain koma, seperti pada kasus Anda yaitu `;`. 
Berikut kode yang dapat digunakan:
```{r} 
cardio <- read.csv("cardio_train.csv", sep= ";")
```

Demikian yang dapat kami sampaikan, semoga membantu.

**Reply by ageljonathan** [reply]
Siap mas Kevin
Thanks
